Compile Error in Linker

```
error[E0425]: cannot find value `input_path` in this scope
  --> crates/concrete_driver/src/linker.rs:63:18
   |
63 |                 &input_path.display().to_string(),
   |                  ^^^^^^^^^^ not found in this scope
```

--- a/crates/concrete_driver/src/linker.rs
+++ b/crates/concrete_driver/src/linker.rs
@@ -60,7 +60,6 @@ pub fn link_binary(objects: &[PathBuf], output_filename: &Path) -> std::io::Resu
             let mut args = vec![
                 "-L/usr/local/lib",
                 "-L/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/lib",
-                &input_path.display().to_string(),
             ];
 
             args.extend(objects.iter().map(|x| x.as_str()));

